### PR TITLE
story continued

### PR DESCRIPTION
**Pull request by shravansonone15**
telling lies no papa added

### CODE REVIEW

File: story1 [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Jony Jony yes papa !
 Eating Suger No Papa!!!
+
+Telling lies no Papa!!!!
```
Comment (sononesharvaree) [inline story1]: please add name of papa if relevant

Comment (sononesharvaree) [reply story1]: please add and update

Comment (shravansonone15) [reply story1]: @sononesharvaree yes addedd please check in new PR

**Review by shravansonone15 — COMMENTED** [review]
ok

**Pull request closed without merging**